client/dist/index.html

```diff
@@ -2,11 +2,13 @@
 <html>
 <head>
   <title>Product App</title>
-</head>
+  <link rel="stylesheet" href="./styles/styles.css">
+  <link rel="stylesheet" href="./styles/qA.css">
+  <script src="https://kit.fontawesome.com/b629c0d2ed.js" crossorigin="anonymous"></script>
```

Comment:
use [google fonts](https://fonts.google.com/) for all your fonts

Reply:
misunderstood what fontawsome was oops, my b

we should find an icon library to standardize however